after/syntax/javascript.vim

```diff
@@ -1,5 +1,5 @@
-if dracula#should_abort('javascript')
-    finish
+if dracula#should_abort('javascript', 'javascriptreact', 'javascript.jsx')
```

Comment:
What if we fixed this function to check all current filetypes (split `&filetype` on a dot)? Then we could maybe not repeat ourselves and say 

```
if dracula#should_abort(..., 'jsx')
```

Reply:
I like this idea.